.changeset/spicy-tomatoes-talk.md

```diff
@@ -0,0 +1,10 @@
+---
+"near-api-js": major
+"@near-js/accounts": major
+"@near-js/biometric-ed25519": major
+"@near-js/crypto": major
+"@near-js/transactions": major
+"@near-js/wallet-account": major
```

Comment:
@gagdiez This caused [packages to be released as 1.0.0](https://github.com/near/near-api-js/pull/1196) release instead of 0.x 😐 

I just wanted to mention it for future reference.